src/mount.coffee

```diff
@@ -315,7 +309,7 @@ class Mounter
     debug "stopping #{@instances.length} instances"
     while @instances.length
       instance = @instances.shift()
-      instance.shutdown()
+      instance.shutdown ->
```

Comment:
Silent empty callbacks are nasty. There should be a comment in the body of the function explaining why this is OK.